src/hooks/use-companies.js

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from 'react-query';
+import { getCompanies } from 'service/company';
+
+const useCompanies = () => {
+  const { data, status } = useQuery('companies', getCompanies);
+  const companies = data?.data;
```

Comment:
It's correct to use the Optional Chaining (= `?`) in this `data` object, but 

- if the first object is `undefined` (i.e. the fetch has not ended), the `companies` variable is going to be `undefined` as well.
- if the data exists (i.e. the fetch has ended), the `companies` variable is going to be an array.

I feel like it makes sense to make the `companies` variable an array in both cases. So that we wouldn't have to use Optional Chaining to use the `.map` in the `companies` variable (`companies?.map`) . Does it make sense?

If so,
```suggestion
  const companies = data?.data || [];
```